### DISCUSSION TITLE
Binding non-QWERTY Keys

### DISCUSSION THREAD

**Discussion by CokeToCode** [helix-editor/helix#2136] [Q&A]
Hi,
I use a German keyboard and would like to make use of my extra keys like 'ä'. Binding them the standard way (ä = "normal_mode" or something alike) doesn't work. Is there any way of making these non-QWERTY bindings work? Sorry if this is a stupid question, I'm a bit of a newbie :).

**Comment by archseer** [comment]
Do you use any modifiers to type that key?

**Reply by Omnikar** [reply]
German keyboard just has ä as an ordinary key, no modifiers.

**Reply by workingj** [reply]
for ä, ü, and ö not on standard german keyboard layouts

**Reply by CokeToCode** [reply]
As Omnikar and workingJ said, it's just an ordinary key.
